org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/commons/TextDocument.java

```diff
@@ -40,22 +41,23 @@ public TextDocument(TextDocumentItem document) {
 	}
 
 	public TextDocument(String text, String uri) {
+		this.lineTracker = new ListLineTracker();
 		super.setUri(uri);
 		super.setText(text);

```

Comment:
change super to this (this.setText(text)) which should fix tests failures (sorry with that!)